app/equipment.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any
+
+
+def knights_parameters(knight_name: Any) -> None:
```

Comment:
The function `knights_parameters` uses `Any` for `knight_name`, which is too generic. Consider using a more specific type, such as `dict`, to better convey the expected structure of `knight_name`.